packages/suite-analytics/src/types/events.ts

```diff
@@ -265,6 +266,12 @@ export type SuiteAnalyticsEvent =
               fiat: string;
           };
       }
+    | {
+          type: EventType.SettingsGeneralChangeBitcoinUnit;
+          payload: {
+              unit: string;
```

Comment:
Should this be typed too?

Reply:
Will be in the future. First, we need to get types out of `suite` package to some `suite-types` package.